recipe/meta.yaml

```
@@ -21,28 +21,28 @@ requirements:
     - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ stdlib("c") }}
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
   host:
     - python
     - pip
+    - setuptools
+    - setuptools_scm
     - numpy
-    - numpy >=1.20.0
   run:
-    - {{ pin_compatible('numpy') }}
     - python
-    - requests >=2.25.0
-    - scipy >=1.7.0
-    - tqdm >=4.60.0
-    - PyYAML >=5.4.0
-    - joblib >=1.0.0
-    - matplotlib-base >=3.5.0
+    - requests >=2.32.3
+    - numpy >=1.26.0
+    - scipy >=1.13.0
+    - tqdm >=4.66.0
+    - PyYAML >=6.0.0
+    - joblib >=1.4.2
+    - matplotlib-base >=3.9.2
     - mne-base >=1.0.0
-    - numba >=0.55.0  # [py<312]
-    - tensorflow >=2.7.0  # [not win and py<312]
-    - wfdb >=3.4.0
-    - edfio >=0.1.1
+    # needs https://github.com/conda-forge/numba-feedstock/pull/152
+    - numba >=0.61.0  # [py<313]
+    - tensorflow >=2.7.0  # [not win and not ppc64le and not aarch64 and py<313]
+    - wfdb >=4.2.0
+    # needs https://github.com/conda-forge/edfio-feedstock/pull/16
+    - edfio >=0.4.4  # [py<313]
```

We could wait a few days to see if https://github.com/conda-forge/sleepecg-feedstock/pull/27 lands, then we don't need this conditional
```suggestion
    # needs https://github.com/conda-forge/edfio-feedstock/pull/16
    - edfio >=0.4.4
```
But if it takes a while we might as well keep it.

Same with `numba` above, I've subscribed to the issue though so easy enough to tick the build number here if needed